spot-webdriver/user/spot-user.js

```diff
@@ -21,6 +21,44 @@ class SpotUser {
         this.reconnectOverlay = new ReconnectOverlay(this.driver);
     }
 
+    /**
+     * Cleanups the session.
+     *
+     * @returns {void}
+     */
+    cleanup() {
+        this.setNetworkOnline();
+        this.disconnectRemoteControlService();
+        this.clearStorage();
+        this.stop();
```

Comment:
No problems were encountered with stop? I had tried doing the same logic as stop before (but also before the manual disconnected of RCS) and the page would get hung.

Reply:
Hmm no, I haven't seen that at least not recently. I've seen that some time ago when closing page after disconnect was done synchronous.